Profiles/README.md

```diff
@@ -1,6 +1,6 @@
 # Profiles
 Fill in the following details about yourself below:
 
-- Your name
-- A picture of yourself
-- A link to your favourite [Pudding article](https://pudding.cool/) or music video
\ No newline at end of file
+- Darya Dyachkova
+- pic of Dasha
```

Comment:
This is meant to be an actual picture of you. On GitHub you can add a pic to an MD file by adding the picture to the folder and then linking it in markdown, like this: ![alt text](img link)